libs/barista-components/micro-chart/src/micro-chart.ts

```diff
@@ -179,6 +180,9 @@ export class DtMicroChart implements OnDestroy {
   }
   private _labelFormatter = (input: number) => input.toString();
 
+  /** Loading text displayed in the internal loading distractor. */
+  @Input('loading-text') loadingText: string;
```

Comment:
```suggestion
  @Input() loadingText: string;
```
I wouldn't name non native attributes.

Reply:
It's called the same in the chart, to keep it consistent I renamed it as well. 
https://github.com/dynatrace-oss/barista/blob/master/libs/barista-components/chart/src/chart.ts#L248-L249